app/views/partials/_itemcard.html.erb

```diff
@@ -2,7 +2,11 @@
     <div class="card itemcard">
         <div class="card-body">
             <div class="image">
-                <%= image_tag(item.image, class: "card-img-top") if item.image.attached? %>
+              <% if item.image.attached? %>
+                <%= image_tag(item.image, class: "card-img-top") %>
+              <% else %>
+                <%= image_tag("image_default.svg", class: "card-img-top filter-gray") %>
```

Comment:
Alternatively we could implement the item the same way as team *br* did it (see [here](https://github.com/hpi-swt2/bookkeeper-portal-blue/blob/dev/app/views/items/new.html.erb#L177-L179) and [here](https://github.com/hpi-swt2/bookkeeper-portal-blue/blob/dev/app/views/items/new.html.erb#L53-L56)).

Reply:
i also thought about the material icon method, probably is the better solution in the manner of asset-amounts. However we will stick the svg for this weekend :D